cli/install/install.go

```diff
@@ -492,36 +578,40 @@ func installTt(binDir string, installCtx InstallCtx, distfiles string) error {
 		}
 	} else {
 		log.Infof("Downloading tt...")
-		err = downloadRepo(search.GitRepoTT, ttVersion, path, logFile, installCtx.verbose)
+		if versionFound {
+			err = downloadRepo(search.GitRepoTT, ttVersion, path, logFile, installCtx.verbose)
+		} else {
+			err = downloadRepo(search.GitRepoTT, "master", path, logFile, installCtx.verbose)
+			util.ExecuteCommand("git", installCtx.verbose, logFile, path, "checkout", ttVersion)
+		}
 	}
 
 	if err != nil {
 		printLog(logFile.Name())
 		return err
 	}
+
 	// Build tt.
 	log.Infof("Building tt...")
-	err = util.ExecuteCommand("mage", installCtx.verbose, logFile, path,
-		"build")
+	err = util.ExecuteCommand("mage", installCtx.verbose, logFile, path, "build")
```

Comment:
Why do we need to change this line? The same question is for 593.

Reply:
fixed